setup.py

```diff
@@ -26,8 +28,14 @@ def reqs(filename):
 TEST_REQS = reqs('test-requirements.txt')
 
 
+with open(os.path.join(os.path.dirname(__file__),
+                       'cloudferry', '__init__.py')) as f:
+    VERSION = re.match(r".*__version__ = '(.*?)'", f.read(), re.S).group(1)
```

Comment:
I'd suggest that we only put version for a github release (git tag), and for github releases `setup.py` should be generated by CI. For `devel` branch we should simply not use version at all